test_common/page_objects/et1/additional_respondents_details_page.rb

```diff
@@ -52,6 +52,7 @@ def remove_another_respondent
 
         def save_and_continue
           page.scroll_to(main_content.save_and_continue_button, align: :bottom)
+          sleep(0.5.second)
```

Comment:
Would be good to have comment as to why this was added.
Something like:
"Save and continue button click randomly fails without half second wait after scrolling"